content/src/service/deployments/types.ts

```diff
@@ -0,0 +1,45 @@
+import { DeploymentFilters, DeploymentSorting, EntityId, EntityType, Pointer, Timestamp } from 'dcl-catalyst-commons'
+import { AuthChain } from 'dcl-crypto'
+import { DeploymentField } from 'src/controller/Controller'
+
+export type PointerChanges = Map<Pointer, { before: EntityId | undefined; after: EntityId | undefined }>
+
+export type DeploymentPointerChanges = {
+  entityType: EntityType
+  entityId: EntityId
+  localTimestamp: Timestamp
+  changes: PointerChanges
+  authChain: AuthChain
+}
+
+export type PointerChangesFilters = Pick<DeploymentFilters, 'from' | 'to' | 'entityTypes'>
+
+export type PartialDeploymentPointerChanges = {
```

Comment:
I'd expect for a partial type to effectively be a part of `DeploymentPointerChanges` but this seems more like a full payload of the API?